src/app/components/ThemeProvider/fontFacesLazy.ts

```diff
@@ -2,26 +2,19 @@ import { Services } from '#app/models/types/global';
 
 const REITH_FONTS_DIR = 'https://static.files.bbci.co.uk/fonts/reith/r2.512/';
 
-const NOTO_SERIF_SINHALA_FONTS_DIR =
-  'https://ws-downloads.files.bbci.co.uk/fonts/NotoSerifSinhala/v1.00/';
+const NOTO_SERIF_SINHALA_FONTS_DIR = 'fonts/NotoSerifSinhala/v1.00/';
```

Comment:
Should we use the full URL here too (and all others)? https://static.files.bbci.co.uk/ws/simorgh-assets/public/fonts

```suggestion
const NOTO_SERIF_SINHALA_FONTS_DIR = 'https://static.files.bbci.co.uk/ws/simorgh-assets/public/fonts/NotoSerifSinhala/v1.00/';
```

Reply:
Yes, I would say that would be safer approach. @SantaZena can you, create a constant to the new location and add it to the URLs?

Reply:
Done it in my last commit.